### DISCUSSION TITLE
npx playwright install应该在哪里执行呀？

### DISCUSSION THREAD

**Discussion by ShawnHu0815** [jia6y/flomo-to-obsidian#28] [Q&A] [answered]
Mac系统，我在Vault中创建了一个Flomo文件夹，在目录中执行了
```
npm init -y
npm install playwright
npx playwright install
```
但是想登录Flomo自动同步时，还是出现了

Flomo Sign-in was failed. Details:
Error: Executable doesn't exist at /Users/shawn/Library/Caches/ms-playwright/chromium-1105/chrome-mac/Chromium.app/Contents/MacOS/Chromium
Looks like Playwright Test or Playwright was just installed or updated. I Please run the following command to download new browsers:

npx playwright install

<3 Playwright Team

请问这个问题怎么解决呀？

**Comment by jia6y** [comment, marked as answer]
抱歉，解决了，卸载插件重新装一下就好了。我之前用的playwright版本比较老。

**Reply by ShawnHu0815** [reply]
已经用上最新版了，谢谢你！很好用的插件！❤️来自中国
